src/H5private.h

```diff
@@ -1979,6 +1979,11 @@ extern hbool_t H5_libterm_g; /* Is the library being shutdown? */
 
 #endif /* H5_HAVE_THREADSAFE */
 
+/* Extern global to determine if we should use selection I/O if available (this
+ * variable should be removed once selection I/O performs as well as the
+ * previous scalar I/O implementation */
+extern hbool_t H5_use_selection_io_g;
```

Comment:
Is this the time to remove this global?

Reply:
Since we haven't implemented MPIO write/read selection yet the selection I/O pathway will be slower for some cases (non-contiguous regular selections), therefore we're still leaving it off by default. We plan to remove the global and always use selection I/O whenever possible before 1.14 though.